app/controllers/assignments_controller.rb

```diff
@@ -108,6 +109,12 @@ def destroy
     end
   end
 
+  def allow_iframe_lti
+    return unless session[:is_lti]
+  
+    response.headers["X-FRAME-OPTIONS"] = "ALLOW-FROM #{session[:lms_domain]}"
```

Comment:
@ayan-biswas0412 , check this once, might apply
https://stackoverflow.com/questions/10205192/x-frame-options-allow-from-multiple-domains